### PR TITLE
Ghbi pcgs performance

### PR DESCRIPTION
**Pull request by bh11**
This is PR #576 which was apparently based upon master. Should be for stable-4.8.

### CODE REVIEW

File: tst/testinstall/mapping.tst [hunk 1]
```diff
@@ -287,6 +287,17 @@ gap> IsGroupHomomorphism(res);
 true
 gap> IsInjective(res);        
 true
+
+# The following test verifies we handle pcgs with large primes in them efficiently.
+# See also GitHub pull request #576
```
Comment (alex-konovalov) [inline tst/testinstall/mapping.tst]: Could you please change the number to 578?

Comment (alex-konovalov) [inline tst/testinstall/mapping.tst]: Or maybe mention both #576 and #578, since the discussion happened in the other one.

**Comment by bh11** [comment]
@alex-konovalov I've added a reference to this PR as well. Since this is a PR against stable-4.8, do you want to merge this when at a good opportunity?

**Comment by alex-konovalov** [comment]
@bh11 thanks - will merge now.

**Comment by fingolfin** [comment]
@bh11 thanks a lot!

**Pull request merged**